src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionUtilityToolFilter.java

```diff
@@ -57,6 +58,9 @@
 
 	private static final String FALSE = "false";
 
+	private static final String BASIC_LTI_LAUNCH_REQUEST = "basic-lti-launch-request";
+	private static final String LTI_MESSAGE_TYPE = "lti_message_type";
+
```

Comment:
These 2 variables being used in sectionsFilter/Servlet and duplicated.... So instead declare one place and use in other class